nasa/earthdata-search
# EDSC-3741: Portal browser image improvements

# Overview

### What is the feature?

Removes the necessity for custom css to display portal images and fixes the Standard Products portal thumbnail.

### What is the Solution?

The portal thumbnail images are now dynamically required. If a portal does not have an image named `logo.png` in the `portals/{portalId}/images` directory, the UI will react will not display an image.

### What areas of the application does this impact?

Portals

# Testing

### Reproduction steps

- **Environment for testing:** Any
- **Collection to test with:** n/a

1. Enter/exit various portals
    - [x] Portal logos are displayed appropriately
    - [x] Standard products thumbnail is displayed when the portal is active
2. View the `example` portal (/search?portal=example)
    - [x] Portal without logo is displayed appropriately
    - [x] No errors are displayed

# Checklist
